src/helpers.tsx

```diff
@@ -76,23 +83,31 @@ export function removeEditorContainers(doc = document): void {
 /**
  * Create an editor portal.
  *
+ * @param {Document} [doc] Document to be used.
  * @param {React.ReactElement} editorElement Editor's element.
+ * @param {Map} editorCache The editor cache reference.
  * @returns {React.ReactPortal} The portal for the editor.
  */
-export function createEditorPortal(editorElement: React.ReactElement): React.ReactPortal {
+export function createEditorPortal(doc = document, editorElement: HotEditorElement, editorCache: Map<Function, React.Component>): React.ReactPortal {
   if (editorElement === null) {
     return;
   }
 
-  const componentName: string = getComponentNodeName(editorElement);
+  const editorContainer = doc.createElement('DIV');
+  const {id, className, style} = getContainerAttributesProps(editorElement.props, false);
 
-  let editorContainer = document.querySelector('#hot-wrapper-editor-container-' + componentName);
-  if (!document.querySelector('#hot-wrapper-editor-container-' + componentName)) {
-    editorContainer = document.createElement('DIV');
-    editorContainer.id = 'hot-wrapper-editor-container-' + componentName;
-    document.body.appendChild(editorContainer);
+  if (id) {
+    editorContainer.id = id;
   }
 
+  editorContainer.className = `${DEFAULT_CLASSNAME} ${className !== void 0 ? className : ''}`;
```

Comment:
We're adding unnecessary space at the end of `class` attr.

```suggestion
  editorContainer.className = [DEFAULT_CLASSNAME, className].join(' ');
```